cmd/getRuleWaiver.go

```diff
@@ -54,13 +54,27 @@ func NewGetRuleWaiverCommand() *cobra.Command {
 				Item{"RESOURCE_ID", *waiver.ResourceID},
 				Item{"RESOURCE_TYPE", *waiver.ResourceType},
 				Item{"RESOURCE_PROVIDER", *waiver.ResourceProvider},
+				Item{"WILDCARD_MODE", *waiver.WildcardMode},
 				Item{"CREATED_AT", format.Unix(waiver.CreatedAt)},
 				Item{"CREATED_BY", waiver.CreatedBy},
 				Item{"CREATED_BY_DISPLAY_NAME", waiver.CreatedByDisplayName},
 				Item{"UPDATED_AT", format.Unix(waiver.UpdatedAt)},
 				Item{"UPDATED_BY", waiver.UpdatedBy},
 				Item{"UPDATED_BY_DISPLAY_NAME", waiver.UpdatedByDisplayName},
 			}
+			var item Item
+			if waiver.ResourceTag != nil {
+				item = Item{"RESOURCE_TAG", *waiver.ResourceTag}
+			} else if *waiver.WildcardMode {
+				item = Item{"RESOURCE_TAG", "*"}
+			} else {
+				// this should never happen
+				item = Item{"RESOURCE_TAG", ""}
+			}
+			// Insert resource-tag after resource-provider
+			index := 11
+			items = append(items[:index+1], items[index:]...)
+			items[index] = item
```

Comment:
If you created the RESOURCE_TAG Item before line 45 you could avoid the slice manipulation here right?